dataeng/resources/expire-vertica-password.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -ex
+
+# Setup
+cd $WORKSPACE/analytics-tools/vertica
+pip install -r requirements.txt
+
+python expire_user_passwords.py \
+    -c $CREDENTIALS \
+    -e $EXCLUDE \
+    -m $MAPPING
```

Comment:
nit: preference to use long form of arguments, e.g. --credentials.

Reply:
Done.